unit_test.sh

```diff
@@ -19,7 +19,7 @@ usql $DB_HOST_URL -c 'drop database if exists bcda_test;'
 usql $DB_HOST_URL -c 'create database bcda_test;'
 usql $TEST_DB_URL -f db/api.sql
 usql $TEST_DB_URL -f db/fixtures.sql
-usql $TEST_DB_URL -f db/test_synthetic_beneficiaries.sql
+usql $TEST_DB_URL -f db/test_synthetic_cclf_files_beneficiaries.sql
```

Comment:
I think we want to intentionally leave this script out, due to the cool stuff happening here:
https://github.com/CMSgov/bcda-app/pull/271/files#diff-12c47e9f6d43cdddab176851f23f31dbR33